pkg/bench/get.go

```diff
@@ -33,11 +33,15 @@ import (
 
 // Get benchmarks download speed.
 type Get struct {
-	CreateObjects int
-	RandomRanges  bool
-	Collector     *Collector
-	objects       generator.Objects
-	Versions      int
+	CreateObjects     int
+	RandomRanges      bool
+	Collector         *Collector
+	objects           generator.Objects
+	Versions          int
```

Comment:
Versions are ignored when using existing objects. 

You could make it...

A) Use up to "Versions" most recent versions when enabled (my preferred solution) or...
B) Error out when set to a value > 1.

Reply:
fixed, versioning has been implemented